.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Don't think we need to custom configure prettier -- these are all default values minus `printWidth` and `printWidth` should be 80 (it's not a hard upper limit)